packages/plans-grid-next/src/components/plan-type-selector/components/interval-type-dropdown.tsx

```diff
@@ -32,8 +32,14 @@ export const IntervalTypeDropdown: React.FunctionComponent< IntervalTypeProps >
 				) : null }
 			</DropdownOption>
 		 ) as unknown as string,
+		accessibleName: option.name as string,
 	} ) );
 
+	const selectedOption = selectOptionsList.find( ( { key } ) => key === supportedIntervalType );
+	const describedByText = selectedOption?.accessibleName
+		? `Currently selected: ${ selectedOption.accessibleName }`
```

Comment:
```suggestion
		? `Currently selected billing period:: ${ selectedOption.accessibleName }`
```
Nitpick: I believe we should add a billing period for further clarity.